crates/ursa-index-provider/src/provider.rs

```diff
@@ -160,7 +160,7 @@ where
 
     fn create_announce_message(&mut self, peer_id: PeerId, mut domain: String) -> Result<Vec<u8>> {
         if domain.is_empty() {
-            domain = "/ip4/127.0.0.1/tcp/8070".to_string();
+            domain = "/ip4/127.0.0.1/tcp/4069".to_string();
```

Comment:
@theBeardA

Reply:
yea, this should be default index provider port and not server port

Reply:
You mean this should not be 4069 e.g. the RPC server port? 

hmm when the gateway queries the cache nodes, it needs to do it at the endpoint with this port. @scott-dn comments?

Reply:
@kckeiks should be the advertisement address:
> https://github.com/ipni/storetheindex/blob/main/api/v0/ingest/schema/schema.ipldsch#L45-L46

Reply:
@theBeardA you're right. This is the wrong place to update the address of where to retrieve the content. Reverted.